include/pybind11/numpy.h

```diff
@@ -213,6 +215,7 @@ struct npy_api {
         API_PyArray_DescrFromScalar = 57,
         API_PyArray_FromAny = 69,
         API_PyArray_Resize = 80,
+        API_PyArray_Newshape = 135,
```

Comment:
Aren't the rest of these in numerical order?

Reply:
Look closely,, they aren't. ;-;

Reply:
Ahh, true. They are in https://github.com/numpy/numpy/blob/623bc1fae1d47df24e7f1e29321d0c0ba2771ce0/numpy/core/code_generators/numpy_api.py

Reply:
@henryiii I fixed them.